public/src/app/platforms.ts

```diff
@@ -13,48 +13,48 @@ export const platforms: PlatformSpec[] = [
     id:'android',
     name:'Android',
     configs:{"alpha": "KeymanAndroid_Build", "beta": "KeymanAndroid_Beta", "stable": "KeymanAndroid_Stable", test: "KeymanAndroid_TestPullRequests"},
-    context: "Test (Keyman - Android)"
+    context: "Keyman - Android"
```

Comment:
Do the `configs` match the TC Build configuration IDs?
1. The consolidated alpha/beta/stable builds have `KeymanAndroid_Build`
2. And the Test-13.0 config is keyman_android_test_13_0 (trigger still a TODO_

Reply:
Yes, the `configs` are supposed to match; I need to do another refactor to support the version-oriented status checks, but for now the changes here support 14.0 build configurations, and I can refactor later.